UIDD/UIDD-Session2/stylesheet.css

```diff
@@ -0,0 +1,357 @@
+body{
```

Comment:
There is a big scope of optimization around this file regarding styles. Also consistency
I believe we might use class for styling purpose as Ids are used more in scripting